perftest/simulations/pom.xml

```diff
@@ -93,6 +93,7 @@
           <skip>${skipITs}</skip>
           <jvmArgs>
             <jvmArg>-Dsim.users=10</jvmArg>
+            <jvmArg>-Dquarkus.http.test-port=${quarkus.http.test-port}</jvmArg>
```

Comment:
Could we simply use `-Dnessie.uri=http://127.0.0.1:${quarkus.http.test-port}` and avoid `.withUri(...)` calls in scala code?

Reply:
I think that the logic for creating the URL should be in the code. Here we want to only inject port. even If I will do `-Dnessie.uri=http://127.0.0.1:${quarkus.http.test-port}`, the scala code needs to concatenate it with `/api/v1` so I would prefer to leave it as is (only changing the port name as in the https://github.com/projectnessie/nessie/pull/3647#discussion_r834565777 comment)

Reply:
> I think that the logic for creating the URL should be in the code.

Actually not. `/api/v1` is, although coded in the Quarkus stuff, rather a configuration option. Similar for `http` vs `https` and the host. Not much work to pass in a full URL in case you wanna test a different endpoint using a manual test or in an external perf test environment.

Reply:
so what's your suggestion?
is it: 
```
    <jvmArg>-Dnessie.uri=http://127.0.0.1:${quarkus.http.test-port}/api/v1</jvmArg>
```
?

Reply:
I'm for `-Dnessie.uri=...`. Does it work?

Reply:
done